int32[] class_names
float32[] bounding_boxes
